src/ol/format/kmlformat.js

```diff
@@ -1011,6 +1030,37 @@ ol.format.KML.readStyle_ = function(node, objectStack) {
 
 
 /**
+ * Reads an array of geometries and creates arrays for common geometry
+ * properties. Then sets them to the multi geometry.
+ * @param {ol.geom.MultiPoint|ol.geom.MultiLineString|ol.geom.MultiPolygon}
+ * multiGeometry
```

Comment:
From the information gathered at the code sprint from @ahocevar, older version of jsdoc had an issue when the parameter name was on a different line. I tested with current version and it is no longer the case. Anyway, thank you for pointing it @elemoine.